receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -1037,6 +1079,302 @@ func TestTranslateV2(t *testing.T) {
+						Metadata: writev2.Metadata{
+							Type: writev2.Metadata_METRIC_TYPE_HISTOGRAM,
+						},
+						Histograms: []writev2.Histogram{
+							{
+								Timestamp:    123456789,
+								Schema:       -53, // CBNH schema
+								Sum:          100.5,
+								Count:        &writev2.Histogram_CountInt{CountInt: 50},
+								CustomValues: []float64{1.0, 2.0, 5.0, 10.0}, // Custom bucket boundaries
+								PositiveSpans: []writev2.BucketSpan{
+									{Offset: 0, Length: 5}, // 5 buckets: 4 custom + 1 overflow
+								},
+								PositiveDeltas: []int64{10, 15, 20, 5, 0}, // Delta counts for each bucket
+							},
+						},
+					},
+				},
+			},
+			expectedStats: remote.WriteResponseStats{
+				Confirmed:  true,
+				Samples:    0,
+				Histograms: 1,
+				Exemplars:  0,
+			},
+			expectedMetrics: func() pmetric.Metrics {
+				metrics := pmetric.NewMetrics()
+				rm := metrics.ResourceMetrics().AppendEmpty()
+				attrs := rm.Resource().Attributes()
+				attrs.PutStr("service.name", "test")
+				attrs.PutStr("service.instance.id", "localhost:8080")
+
+				sm := rm.ScopeMetrics().AppendEmpty()
+				sm.Scope().SetName("OpenTelemetry Collector")
+				sm.Scope().SetVersion("latest")
+				m1 := sm.Metrics().AppendEmpty()
+				m1.SetName("test_cbnh_histogram")
+				m1.SetUnit("")
+				m1.SetDescription("")
+				hist := m1.SetEmptyHistogram()
+				hist.SetAggregationTemporality(pmetric.AggregationTemporalityCumulative)
+
+				dp := hist.DataPoints().AppendEmpty()
+				dp.SetStartTimestamp(pcommon.Timestamp(123456000 * int64(time.Millisecond)))
+				dp.SetTimestamp(pcommon.Timestamp(123456789 * int64(time.Millisecond)))
+				dp.SetSum(100.5)
+				dp.SetCount(50)
+				dp.ExplicitBounds().FromRaw([]float64{1.0, 2.0, 5.0, 10.0})
+				dp.BucketCounts().FromRaw([]uint64{10, 15, 20, 5, 0}) // Delta counts
+
+				return metrics
+			}(),
+		},
+		{
+			name: "CBNH translation with stale NaN",
+			request: &writev2.Request{
+				Symbols: []string{
+					"",
+					"__name__",
+					"test_cbnh_histogram_stale",
+					"job",
+					"test",
+					"instance",
+					"localhost:8080",
+					"seconds",
+					"Test CBNH histogram with stale NaN",
+				},
+				Timeseries: []writev2.TimeSeries{
+					{
+						LabelsRefs: []uint32{1, 2, 3, 4, 5, 6}, // __name__=test_cbnh_histogram_stale, job=test, instance=localhost:8080
+						Metadata: writev2.Metadata{
+							Type: writev2.Metadata_METRIC_TYPE_HISTOGRAM,
+						},
+						Histograms: []writev2.Histogram{
+							{
+								Timestamp:    123456789,
+								Schema:       -53, // CBNH schema
+								Sum:          math.Float64frombits(value.StaleNaN),
+								Count:        &writev2.Histogram_CountInt{CountInt: 50},
+								CustomValues: []float64{1.0, 2.0, 5.0, 10.0}, // Custom bucket boundaries
+								PositiveSpans: []writev2.BucketSpan{
+									{Offset: 0, Length: 5}, // 5 buckets: 4 custom + 1 overflow
+								},
+								PositiveDeltas: []int64{10, 15, 20, 5, 0}, // Delta counts for each bucket
```

Comment:
It  would be pretty weird to send positive count+occupied buckets with a stale sample. I would not expect this. 
```suggestion
								Count:        &writev2.Histogram_CountInt{CountInt: 0},
								CustomValues: []float64{1.0, 2.0, 5.0, 10.0}, // Custom bucket boundaries
```